language/move-vm/runtime/src/interpreter.rs

```diff
@@ -780,7 +930,7 @@ impl<'txn> Interpreter<'txn> {
         context: &mut dyn InterpreterContext,
         address: AccountAddress,
         idx: StructDefinitionIndex,
-        type_actuals_idx: LocalsSignatureIndex,
+        ty_arg_sigs: &Signature,
```

Comment:
I'll look into that however keep in mind that creating a Signature amount to nothing. It should be a noop.
And I mention also because it's not clear to me if those should all be around &Signatures?
don't know, thoughts?